config/lobby/db/003_add_bcrypt_password_column

```diff
@@ -0,0 +1,17 @@
+start transaction;
+
+alter table ta_users add column bcrypt_password character(60) check (char_length(bcrypt_password)=60);
+alter table ta_users alter column password drop not null;
```

Comment:
Would it make more sense to call this column `md5crypt_password` so it's clear why it's different from `bcrypt_password` without having to look at the comments?  Then, in the future, when we merge the two password columns back into a single column, the `bcrypt_password` column would just be renamed to `password`.

Reply:
We could do that, but in this case we'd need to change all references in the code to that as well, and change everything back afterwards...
Merging this back at a later time is just one additional statement:
```
UPDATE ta_users SET password=bcrypt_password;
ALTER TABLE ta_users REMOVE COLUMN bcrypt_password;
```

Reply:
That was one of the reasons I suggested we extract the password-related column names to constants because they were being used in so many places and would be a pain to update.  NBD though. :smile: